pkg/capr/planner/instructions.go

```diff
@@ -85,19 +85,25 @@ func (p *Planner) generateInstallInstruction(controlPlane *rkev1.RKEControlPlane
 // from the installer image based on the control plane configuration. It will generate a restart stamp based on the
 // passed in configuration to determine whether it needs to start/restart the service being managed.
 func (p *Planner) addInstallInstructionWithRestartStamp(nodePlan plan.NodePlan, controlPlane *rkev1.RKEControlPlane, entry *planEntry) (plan.NodePlan, error) {
```

Comment:
The function description is outdated. Could you update it?

Reply:
Updated.